src/test/java/de/zalando/aruha/nakadi/validation/RFC3339DateTimeValidatorTest.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.validation;
+
+import org.junit.Test;
+
+import static de.zalando.aruha.nakadi.utils.IsOptional.isAbsent;
+import static de.zalando.aruha.nakadi.utils.IsOptional.isPresent;
+import static org.junit.Assert.assertThat;
+
+public class RFC3339DateTimeValidatorTest {
+    private final RFC3339DateTimeValidator validator = new RFC3339DateTimeValidator();
+
+    @Test
+    public void requireMetadataOccurredAtToBeFormattedAsDateTime() {
+        final String invalidDateTimes[] = new String[] {
+                "x", // totally non sense string
+                "1996-10-15T16:39:57+07:00:30", // invalid seconds in the offset
+                "1996-10-15T16:39:57+0700", // invalid missing colon in the offset
+                "1996-10-15T16:39:57 07:00", // invalid missing signal in the offset
+                "1996-10-45T16:39:57Z", // check for lenience (there are no months with 45 days)
+                "1996-10-45 16:39:57Z", // requires "T" as separator
+                "1996-10-45t16:39:57Z", // the RFC requires uppercase T, sorry
+                "1996-10-45T16:39:57z", // the RFC requires uppercase Z as well, sorry again
+                "1996-10-45T16:39:57", // offsets are required
+        };
+
+        final String validDateTimes[] = new String[] {
+                "1996-10-15T16:39:57+07:00", // just a very simple example
+                "1996-10-15T16:39:57-07:00", // just a very simple example
+                "1996-10-15T16:39:57.123+07:00", // simple example with milliseconds
+                "1996-10-15T16:39:57.1234Z", // tricky 4 milliseconds digits (yes it's valid, sorry)
```

Comment:
`ISO_OFFSET_DATE_TIME` allows anywhere from 0-9 fractional digits so it might make sense to add tests for all of them.